Update blacklist

kylerobi opened the pull request.

kylerobi committed:
Update blacklist

Flaminel commented on blacklist:
```diff
@@ -845,6 +847,8 @@
 *.zw
 *.zwi
 *.zz
+Extra*
```
There are quite a few movies and shows starting with this string, such as `Extraordinary Siamese Story` and `Extra`. This would block those items because of this.

Flaminel commented on blacklist:
```diff
@@ -845,6 +847,8 @@
 *.zwi
 *.zz
+Extra*
+Featurettes*
```
Only files are currently checked against the blacklist, so directories are not going to be removed. Is this still relevant here?

kylerobi replied:
I was unaware of this behavior. I'll take this out

Flaminel commented on blacklist:
```diff
@@ -845,6 +847,8 @@
 VOSTFR
-api
\ No newline at end of file
+api
```
You've added a new line here. Did you test to check if this affects the behavior in any way? As far as I remember, empty lines are not expected here.

kylerobi committed:
Remove .srt and Extra*

aligning it with the original branch while keeping ".srt" out

kylerobi committed:
adding *.m2ts & Featurettes*

adding *.m2ts & Featurettes* ti blocklist as these tend to not have relevant files for Jellyfin and require manual failing to find another file

kylerobi committed:
Remove 'Featurettes*' and 'Trailer.*' from blacklist

kylerobi committed:
Consolidate blacklist entries for r0*

kylerobi committed:
Update blacklist by removing certain file patterns

Removed specific patterns for file types from blacklist.
